opentelemetry-sdk/src/opentelemetry/sdk/logs/__init__.py

```diff
@@ -111,6 +113,48 @@ def force_flush(self, timeout_millis: int = 30000):
         """
 
 
+class OTLPHandler(logging.Handler):
```

Comment:
Should this be called an emitter rather than a handler? Or is this a convenience handler to allow users to add this handler to their python root handler?

Reply:
Yes, this is one the the handlers users can attach to root logger. I believe the name Handler makes more sense here.